ecf/cam/stats/jevs_hrrr_grid2obs_stats.ecf

```diff
@@ -3,7 +3,7 @@
 #PBS -j oe
 #PBS -S /bin/bash
 #PBS -q dev
-#PBS -A VERF-DEV
+#PBS -A EVS-DEV
```

Comment:
Change to VERF-DEV?

Reply:
Yes please change this from "EVS-DEV" to "VERF-DEV".

Reply:
Change is made.